Lombiq.Tests.UI/Extensions/NavigationUITestContextExtensions.cs

```diff
@@ -74,8 +74,24 @@ await context.Configuration.Events.AfterNavigation
 
     public static string GetCurrentAbsolutePath(this UITestContext context) => context.GetCurrentUri().AbsolutePath;
 
-    public static Uri GetAbsoluteUri(this UITestContext context, string relativeUrl) =>
-        new(context.Scope.BaseUri, relativeUrl.TrimStart('/'));
+    public static Uri GetAbsoluteUri(this UITestContext context, string relativeUrl)
+    {
+        var localPath = context.Scope.BaseUri.LocalPath;
+        var splitRelativeUrl = relativeUrl.TrimStart('/').Split('?');
+        var query = splitRelativeUrl.Length > 1 ? splitRelativeUrl[1] : string.Empty;
+        var path = splitRelativeUrl[0];
```

Comment:
The query and path are also available in `context.Scope.BaseUri.AbsolutePath` and `context.Scope.BaseUri.Query`. Why is this logic, and using `LocalPath`, necessary?